src/view/components/cell/cell.handler.js

```diff
@@ -14,26 +14,49 @@ class CellHandler extends Component {
 		const model = this.root.model;
 		const table = this.root.table;
 		const element = this.element;
-		model.focusChanged.on(e => {
-			if (e.hasChanges('rowIndex') || e.hasChanges('columnIndex')) {
-				const cell = table.body.cell(e.state.rowIndex, e.state.columnIndex);
-				const cellModel = cell.model();
-				if (cellModel) {
-					element.classList.add('q-grid-active');
-					cell.addClass('q-grid-animate');
-					const target = cell.element;
+
+		// When navigate first or when animation wasn't applied we need to omit
+		// next navigation event to make handler to correct position.
+		let isValid = false;
+		model.navigationChanged.watch(e => {
+			if (e.hasChanges('cell')) {
+				const domCell = table.body.cell(e.state.rowIndex, e.state.columnIndex);
+				const cell = e.state.cell;
+				if (cell) {
+					const oldColumn = (e.changes.cell.oldValue || {}).column || {};
```

Comment:
Confused a bit with these lines

Reply:
oldValue can be null so to get something not nullable to oldColumn we write like that

Reply:
any other ideas?

Reply:
@klumba12 Maybe 
`` e.changes.cell.oldValue ? e.changes.cell.oldValue.column : {} ``